scss/tech.scss

```diff
@@ -0,0 +1,48 @@
+.tech-border {
+  border: 1px solid $earls-green;
+  border-radius: 5px;
+  height: 300px;
+  width: 200px;
+}
+
+.tech-content {
+  background-color: $earls-green;
+  left: 10px;
+  font-family: 'Roboto', sans-serif;
+  position: relative;
+  top: 10px;
+}
+
+.tech-logo {
+  height: 100px;
+  left: 25%;
+  position: absolute;
```

Comment:
you don't have to use position absolute in every element inside of the tech and move it around with top and left values. you should use a column flex layout and let the elements adapt to this layout.